platform/ios/os_ios.mm

```diff
@@ -181,7 +177,9 @@ void register_dynamic_symbol(char *name, void *address) {
 }
 
 void OS_IOS::start() {
-	Main::start();
+	if (Main::start()) {
```

Comment:
Approved too fast, this might actually fail. I think it should be
```suggestion
	if (Main::start() == EXIT_SUCCESS) {
```
following https://github.com/godotengine/godot/pull/89229